MetaMorpheus/TaskLayer/GPTMDTask/GPTMDTask.cs

```diff
@@ -114,7 +114,9 @@ protected override MyTaskResults RunSpecific(string OutputFolder, List<DbForTask
                 .GroupBy(b => new Tuple<string, int, double?>(b.FullFilePath, b.ScanNumber, b.PeptideMonisotopicMass))
                 .Select(b => b.First()).ToList();
 
-            new FdrAnalysisEngine(allPsms, tempSearchMode.NumNotches, CommonParameters, this.FileSpecificParameters, new List<string> { taskId }).Run();
+            new FdrAnalysisEngine(allPsms, tempSearchMode.NumNotches, CommonParameters, this.FileSpecificParameters, new List<string> { taskId }, doPEP:false).Run();
+
+            allPsms = allPsms.Where(p => p.FdrInfo.QValueNotch < 0.05).ToList();
```

Comment:
I'd recommend stating the `0.05` cutoff hardcoded here as a named constant at the top of this file.

Reply:
Could note in a comment for the constant we chose this value (to limit the bulk of PSMs to process).